### DISCUSSION TITLE
Backfill BigQuery data firestore collectionGroup - Error: Value for argument "desiredPartitionCount" is not a valid integer.

### DISCUSSION THREAD

**Discussion by bicced** [firebase/extensions#1413] [Q&A] [answered]
I am receiving this error when attempting to backfill my bigquery table with previous data from collectiongroup. I cannot find a solution to this, please help! <3

I was following this import guide: https://github.com/firebase/extensions/blob/master/firestore-bigquery-export/guides/IMPORT_EXISTING_DOCUMENTS.md

{"severity":"INFO","message":"View with id Orders_raw_changelog_raw_latest already exists in dataset firestore_export."}
Importing data from Cloud Firestore Collection (via a Collection Group query): Orders, to BigQuery Dataset: firestore_export, Table: Orders_raw_changelog_raw_changelog
Error importing Collection to BigQuery: Error: Value for argument "desiredPartitionCount" is not a valid integer.

**Comment by jauntybrain** [comment, marked as answer]
This should be now fixed with #1930.
